runtime/src/accounts.rs

```diff
@@ -1303,24 +1297,6 @@ impl Accounts {
                     );
 
                     if execution_status.is_ok() || is_nonce_account || is_fee_payer {
-                        if !preserve_rent_epoch_for_rent_exempt_accounts
-                            && account.rent_epoch() == INITIAL_RENT_EPOCH
-                        {
-                            let rent = rent_collector
```

Comment:
@behzadnouri 
This PR looks like it is correctly removing the feature.
I am not entirely sure what this code here is doing. Is this code gone because we aren't allowed to create rent paying accounts anymore so the only side effect was updating `rent_epoch`?

Reply:
for posterity, from behzad:

that branch was added separately in:
https://github.com/solana-labs/solana/pull/26851
the PR has some context